tests/integration/test_vulnerability_detector/test_windows/test_cpe_indexing.py

```diff
@@ -157,14 +96,14 @@ def mock_system(request):
     control_service('stop', daemon='wazuh-db')
 
     # Clean tables
-    vd.clean_vd_tables(agent='000')
+    vd.clean_vd_tables(agent=mock_agent)
 
     control_service('start', daemon='wazuh-db')
 
 
 def test_window_version_indexing(get_configuration, configure_environment, restart_modulesd, check_cve_db, mock_system):
     wazuh_log_monitor.start(
```

Comment:
After talking with @snaow we have considered that new development or changes into existing tests should follow new repository standards about Wazuh daemons. For now, it is required to use fixtures to start only required daemons, and turn them off when the test is over.  That means, that the default status of Wazuh would be off and each test should be responsible for daemons handling.
The reasons for this change are:
- Faster testing
- More encapsulated tests